src/elements/content-sidebar/versions/VersionsSidebar.js

```diff
@@ -49,8 +61,26 @@ const VersionsSidebar = ({ error, isLoading, parentName, versions, ...rest }: Pr
         >
             <LoadingIndicatorWrapper className="bcs-Versions-content" crawlerPosition="top" isLoading={isLoading}>
                 {showError && (
-                    <InlineError title={<FormattedMessage {...messages.versionServerError} />}>
+                    <InlineError
+                        title={
+                            errorUpsell ? (
+                                <FormattedMessage {...messages.versionServerErrorUpsell} />
+                            ) : (
+                                <FormattedMessage {...messages.versionServerError} />
+                            )
+                        }
+                    >
                         <FormattedMessage {...error} />
+                        {errorUpsell && (
```

Comment:
Can we standardize on either "upgrade" (preferred) or "upsell"?

Reply:
I am renaming my variables to make them clearer. We have the upgrade button, which should really be the only place I use upgrade. and then the rest is an upsell, which is asking people to update.